pages/index.tsx

```diff
@@ -22,23 +39,37 @@ const Index: NextPage<Props> = props => {
   };
 
   return (
-    <Container maxWidth="lg" className="flex justify-between pt-4">
-      <Typography variant="h1" className="text-3xl">
-        dev-directory
-      </Typography>
-      <Box className="flex items-center gap-4">
-        <Typography>
-          You are {!isAuthed ? 'not' : ''} logged in
+    <Container maxWidth="lg" className="pt-4">
+      <Box className="flex justify-between">
+        <Typography variant="h1" className="text-3xl">
+          dev-directory
+        </Typography>
+        <Box className="flex items-center gap-4">
+          <Typography>
+            You are {!isAuthed ? 'not' : ''} logged in
+          </Typography>
+          {isAuthed ? (
+            <Button variant="contained" color="secondary" onClick={onLogout}>
+              Log out
+            </Button>
+          ) : (
+            <Button variant="contained" color="primary" href={getDiscordOauthUrl()}>
+              Log in
+            </Button>
+          )}
+        </Box>
+      </Box>
+      <Box sx={{ pt: 4 }}>
```

Comment:
Is there a reason this uses the `sx` prop vs the tailwind class name (as is used in the `Container` component above)?